core/templates/base-components/base-content.directive.ts

```diff
@@ -19,7 +19,7 @@
 require('base-components/loading-message.component.ts');
 require('base-components/warnings-and-alerts.directive.ts');
 require('pages/OppiaFooterDirective.ts');
-
+require('domain/bottom_navbar/bottom-navbar-status.service.ts');
```

Comment:
Add newline between directive and service imports.

Reply:
DOne